admin/src/pages/View/components/NavigationItemForm/index.tsx

```diff
@@ -221,7 +221,7 @@ const NavigationItemForm: React.FC<NavigationItemFormProps> = ({
       return isString(title) && !isEmpty(title) ? await slugify(title).then(prop("slug")) : undefined;
     } else if (related) {
       const relationTitle = extractRelatedItemLabel({
-        ...contentTypeEntities.find(_ => _.id === related),
+        ...contentTypeEntities.find(_ => String(_.id) === related),
```

Comment:
`_.id.toString() === related.toString()`

to be sure about type of both

Reply:
Wouldn't it be safer to use `String(...)`? The function parameters of `generateUiRouterKey` defined `related?: string` so it could potentially be undefined. Calling `.toString()` on that would obviously not work.